app/views/RoomView/index.js

```diff
@@ -724,6 +725,20 @@ class RoomView extends React.Component {
 		}
 	});
 
+	getLock = (wasPlaying) => {
```

Comment:
Let's say we want to allow to watch a video without having to fullscreen.
IMO we could have functions `playMedia`, `pauseMedia` and `stopMedia`.
All of them would control media being played on `RoomView`, being audio or video.

Reply:
Here my only doubt is that both the pauseMedia and stopMedia will at the end perform the same task i.e should I create just one or have both

Reply:
You're right. Play and stop then.